packages/@react-stately/numberfield/src/useNumberFieldState.ts

```diff
@@ -60,6 +60,10 @@ export interface NumberFieldState {
 }
 
 interface NumberFieldStateProps extends NumberFieldProps {
+  /**
+   * The locale that should be used for parsing.
+   * @default 'en-US'
+   */
   locale: string
```

Comment:
Why is this change part of this pr?

Reply:
happened to notice it while working on useTabListState docs and was comparing it to useNumberFieldState docs